models/wiki.go

```diff
@@ -21,44 +21,6 @@ import (
 	"github.com/gogits/gogs/modules/setting"
 )
 
-// workingPool represents a pool of working status which makes sure
```

Comment:
This was moved out of wiki so that any repo, not just wiki repos, could be have a workingPool. It was moved to working_pool.go for more global use.